.gitlab-ci.yml

```diff
@@ -87,12 +83,9 @@ e2e-tests-dev:
   artifacts:
     paths:
       - ./env.json
+      - ./sv_kubeconfig_content.yaml
```

Comment:
Why is this needed?

Reply:
sv_kubeconfig_content.yaml contains the kubeconfig content for the borrowed testbed
to pass the content from one stage to another we make use of the artifacts
any file required to be part of artifacts, the paths should be mentioned here.